javascript/textareas.js

```diff
@@ -373,29 +367,29 @@ function localMessageHandler(msg, port) {
         var textarea_observer = new MutationSummary({
             callback: handleUpdatedElements,
             queries: [
-                {   // we don't want the source page accidently duplicating our tags
-                    element: "img[class='ewe_edit_button']"
+                {
+                    // we don't want the source page accidently duplicating our tags
+                    element: "img[class='ewe_edit_button']",
                 },
                 {
-                    element: "textarea"
```

Comment:
js prettier automatically did that change. I would guess it only changes to singlequote when there's no singlequote in the string itself so it must not escape.